clang/include/clang-c/Dependencies.h

```diff
@@ -125,6 +125,17 @@ CINDEX_LINKAGE void
 clang_experimental_DependencyScannerServiceOptions_setCASOptions(
     CXDependencyScannerServiceOptions Opts, CXCASOptions);
 
+/**
+ * Turn on working directory optimization. The dependency scanner service
+ * option Opts will indicate to the scanner that the current
+ * working directory can be ignored when computing the pcms' context hashes.
+ * The scanner will then determine if it is safe to optimize each module and act
+ * accordingly.
+ */
+CINDEX_LINKAGE void
+clang_experimental_DependencyScannerServiceOptions_enable_cwd_optimization(
```

Comment:
To be consistent I think this should be `clang_experimental_DependencyScannerServiceOptions_setCWDOptimization(CXDependencyScannerServiceOptions , int)`.